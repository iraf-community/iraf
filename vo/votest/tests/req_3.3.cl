#
#  Registry query in multiple ways.

# Set the test description string.
votest.descr = "Registry query in multiple ways."

print ("------------------------------------------------------------------")
print ("Req  3.3:  Users shall be able to refer to a VO Resource (in tasks")
print ("           requiring a Resource parameter) in multiple ways:")
print ("------------------------------------------------------------------")


fcache init


# Execute the test commands.
print ("This requirement is demonstrated by the sub-tests.\n")
