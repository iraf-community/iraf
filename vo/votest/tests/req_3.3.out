------------------------------------------------------------------
Req  3.3:  Users shall be able to refer to a VO Resource (in tasks
           requiring a Resource parameter) in multiple ways:
------------------------------------------------------------------
This requirement is demonstrated by the sub-tests.

